analyzer-comments/javascript/elyses-analytic-enchantments/must_avoid_imperative_loop.md

```diff
@@ -0,0 +1,7 @@
+# must avoid imperative loop
+
+Avoid use of for loops with this practice exercise.
```

Comment:
The exercise this comment is for is a learning exercise, not a practice exercise. Either use `learning exercise` or just `exercise`.